src/NuGet.Clients/NuGet.CommandLine/Program.cs

```diff
@@ -130,6 +130,8 @@ public static int MainCore(string workingDirectory, string[] args)
                     console.WriteLine(LocalizedResourceManager.GetString("InvalidArguments"), commandName);
 
                     p.HelpCommand.ViewHelpForCommand(commandName);
+
+                    throw new ExitCodeException(1);
```

Comment:
I would do `return 1;` similar to what's being done in https://github.com/NuGet/NuGet.Client/pull/2505/files#diff-c067a9e9f3d8e725248a12832512a4aaR160

Reply:
Let's try with `return 1;`